assets/wizards/updates/index.js

```diff
@@ -0,0 +1,60 @@
+import '../../shared/js/public-path';
+
+/**
+ * Updates
+ */
+
+/**
+ * WordPress dependencies.
+ */
+import { Component, render, Fragment, createElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Material UI dependencies.
+ */
+import HeaderIcon from '@material-ui/icons/Widgets';
+
+/**
+ * Internal dependencies.
+ */
+import { withWizard } from '../../components/src';
+import Router from '../../components/src/proxied-imports/router';
+import { DevInfo } from './views';
+
+const { HashRouter, Redirect, Route, Switch } = Router;
+
+class UpdatesWizard extends Component {
+	/**
+	 * Render
+	 */
+	render() {
+		const { pluginRequirements } = this.props;
+		return (
+			<Fragment>
```

Comment:
I don't think we need `Fragment` here since there's only one child.

Reply:
Done in bbd8ce88b6765d22ecc9ebd311e4262f78bc9a69